buildscripts/ci/translation/s3_packandsend.py

```
@@ -58,19 +63,19 @@ def processTsFile(prefix, langCode, data):
+            data[langCode][prefix] = {}
 
-        data[lang_code][prefix]["file_name"] = filename + ".qm"
-        data[lang_code][prefix]["hash"] = str(hash_file.hexdigest())
-        data[lang_code][prefix]["file_size"] = file_size
+        data[langCode][prefix]["file_name"] = filename + ".qm"
+        data[langCode][prefix]["hash"] = str(hash_file.hexdigest())
+        data[langCode][prefix]["file_size"] = file_size
 
-        return True
+        return True, True
     else:
```

Ruff (0.15.10)

[warning] Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)